app/controllers/HonestyDeclarationController.scala

```diff
@@ -41,6 +43,9 @@ class HonestyDeclarationController @Inject()(honestyDeclarationView: honesty_dec
   }
 
   val submit: Action[AnyContent] = isAuthenticatedWithProfile() {
-    _ => _ => Future.successful(Redirect(applicantRoutes.IncorpIdController.startIncorpIdJourney()))
+    implicit request =>
+      implicit profile =>
+        vatRegistrationService.submitHonestyDeclaration(regId = profile.registrationId, honestyDeclaration=true)
```

Comment:
Check the formatting please

Reply:
This always submits true. It should be passed the value from the form and store that.